### PR TITLE
Set the level of the Ltac2 "now" notation to 6.

### LINKED ISSUE #15122: Ltac2: "now" tactic does not work in 8.14.0 (did in 8.13.2)
#### Description of the problem

In 8.14.0 the `now` tactic appears to be broken in Ltac2. It did work as expected in 8.13.2.

Here is an example which does work in 8.13.2 but not in 8.14.0:

```
Require Import Ltac2.Ltac2.
Require Import ZArith.
Require Import Reals.

Lemma Ltac2_now_regression : forall i:Z,
  (0 <= i <= 4294967295)%Z ->
  (0 <= IZR i <= 4294967295)%R.
Proof.
  intros i H.
  now split; apply IZR_le.
Qed.
```

In 8.14.0 this works:

```
Proof.
  intros i H.
  split; apply IZR_le; apply H.
Qed.
```

### PR DESCRIPTION
**Pull request by ppedrot**
Fixes #15122: Ltac2: "now" tactic does not work in 8.14.0.

### CODE REVIEW

**Review by MSoegtropIMC — APPROVED** [review]
Looks good to me - thanks!

**Comment by Zimmi48** [comment]
Probably deserves a changelog entry.

**Comment by Zimmi48** [comment]
@MSoegtropIMC will you self-assign?

**Comment by SkySkimmer** [comment]
Please hurry this up if you want it in 8.15.
Also could use a changelog.

**Comment by MSoegtropIMC** [comment]
Sorry, I am busy with Coq Platform. I will merge it today evening.

**Comment by MSoegtropIMC** [comment]
@ppedrot : do you want to add a changelog entry?

**Comment by ppedrot** [comment]
OK, if you want.

**Comment by MSoegtropIMC** [comment]
> OK, if you want.

as @SkySkimmer indicated it is a change in a user facing feature, so IMHO it should have a changelog entry.

**Comment by MSoegtropIMC** [comment]
Sorry, I just saw that you already did it. I will merge as soon as PR passes.

**Comment by ppedrot** [comment]
(The fiat-crypto CI failure is unrelated.)

**Comment by MSoegtropIMC** [comment]
The fiat crypto CI failed with "Fatal error: out of memory" which I guess is unrelated.

Unless someone objects, I will merge in 1 hour.

**Comment by ppedrot** [comment]
@MSoegtropIMC I think you closed this by error, I'm reopening it thus.

**Comment by MSoegtropIMC** [comment]
@ppedrot : yes, sorry, thanks!

There is a message that coqbot removed it from 8.14.2 which might be due to the erroneous closing - should I add it back?

**Comment by ppedrot** [comment]
Yes, please add it back indeed before merging.

**Comment by MSoegtropIMC** [comment]
Did you restart CI? Should I wait for it?

**Comment by ppedrot** [comment]
I haven't restarted it manually at least, maybe it's automatic after a reopen?

**Comment by MSoegtropIMC** [comment]
OK, I will merge in 40 minutes then.

**Comment by MSoegtropIMC** [comment]
@ppedrot : I decided to wait for the CI run to finish.

**Comment by Zimmi48** [comment]
fiat-crypto is failing everywhere, so there is no hope to get it to succeed here by just restarting the job

**Pull request merged**
